### PR TITLE
Added AuthorBio on single post

### PR DESCRIPTION
**Pull request by nil1511**

### CODE REVIEW

File: src/post/AuthorBio.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    steemdb.accounts({
+      account: this.props.authorName
+    }, (err, result) => {
+      if (result.length) {
+        const author = result[0];
+
+        try {
+          author.json_metadata = JSON.parse(result[0].json_metadata);
+        } catch (e) {
+          author.json_metadata = {};
```
Comment (p0o) [inline src/post/AuthorBio.js]: better to throw an error on catch:

```
throw new Error(`Error parsing jsonMetadata for user ${username}`);
```

File: src/post/AuthorBio.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+import Avatar from '../widgets/Avatar';
+import Loading from '../widgets/Loading';
+import Follow from '../widgets/Follow';
+import { followUser, unfollowUser, getFollowing } from '../user/userActions';
+
+@connect(
+  ({ auth, userProfile }) => ({ auth, following: userProfile.following }),
+  dispatch => bindActionCreators({ followUser, unfollowUser, getFollowing }, dispatch)
+)
+
```
Comment (p0o) [inline src/post/AuthorBio.js]: better to not leave an empty line after any HoC, these are not separated parts.

File: src/post/AuthorBio.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+class AuthorBio extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      author: null,
+    };
+  }
+
+  componentDidMount() {
+    const loggedInUser = this.props.auth.user.name;
```
Comment (p0o) [inline src/post/AuthorBio.js]: const loggedInUser = this.props.auth.user ? this.props.auth.user.name : '';

**Comment by nil1511** [comment]
@p0o Thanks. I have made the changes accordingly.

**Pull request merged**